addok/helpers/serializers.py

```diff
@@ -0,0 +1,14 @@
+import json
+import marshal
+import zlib
+
+
+class Compressed:
```

Comment:
Why isn't it named `ZLibSerializer`? What about inheriting from a `BaseSerializer` which implements `dumps/loads` and raises `NotImplementedError` accordingly? It adds a bit of complexity but it's closer to community standards[ref. needed] :)